dts/bindings/fuel-gauge/battery_cutoff.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS compliant or
+  similarly smart battery fuel gauges.
+
+  Note: These properties are to be used with meaningful defaults in fuel gauge ICs that implement
+  battery cutoff support. See the default fuel gauge SBS Gauge compatible as an example.
+
+properties:
+  battery_cutoff_support:
+    description: |
+      Helper prop that indicates whether this device can cutoff the battery, this is also often
+      referred to as ship or sleep mode. Any non-zero value means battery cutoff is supported.
+    type: int
+  battery_cutoff_reg_addr:
+    description: |
+       Address of register to receive cutoff payload for battery cutoff.
+    type: int
+  battery_cutoff_payload:
+    description: |
+       Payload to write to cutoff battery. This is often a sequence of two duplicate values.
```

Comment:
```suggestion
       Payload to write to cutoff battery register. This must be an array of 2 integers.  
       If your battery only requires a single payload, duplicate the value.  This will 
       cause the driver to issue the cutoff twice.
```